Client/Connection.php

```diff
@@ -150,13 +151,17 @@ public function setBulkParams(array $params)
 
     /**
      * Flushes the current query container to the index, used for bulk queries execution.
+     *
+     * @throws EmptyBulkException
      */
     public function commit()
     {
+        if (!$this->bulkQueries) {
+            throw new EmptyBulkException('Nothing persisted to commit.');
```

Comment:
Right message would be: `Bulk container is empty. Add some queries before commit.`